data/service/service/standard.go

```diff
@@ -310,27 +308,23 @@ func getenvStr(key string) (string, error) {
 	return v, nil
 }
 
-// Retrieve the PUSH_TO_READ_STORE_ENABLED env variable
-// It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False.
-// Any other value returns true by default.
-func getPushToReadStoreEnv() bool {
-	s, err := getenvStr("PUSH_TO_READ_STORE_ENABLED")
+func getArchivedDataTypesEnv() []string {
+	s, err := getenvStr("WRITE_TO_ARCHIVE")
```

Comment:
I would keep the variable as it was:
* ARCHIVED_DATA_TYPES
* BUCKETED_DATA_TYPES
* LEGACY_DATA_TYPES

new names look like booleans (WRITE_TO_ARCHIVE=true/false)